addons/disable-stage-drag-select/addon.json

```diff
@@ -16,9 +20,21 @@
       "matches": ["projects"]
     }
   ],
+  "settings": [
+    {
+      "name": "Allow dragging while project is stopped",
+      "id": "drag_while_stopped",
+      "type": "boolean",
+      "default": false
```

Comment:
I think this should be true by default, but this is fine too.

Reply:
The ideal default setting vs. backwards compatibility dilemma...

Reply:
I think keeping the default as off is fine since you can hold Shift to drag, no matter if the project is running or not!